src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_eckey.h

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
+
+#include "pal_types.h"
+#include "pal_compiler.h"
+#include "pal_jni.h"
+
+typedef struct EC_KEY
```

Comment:
Typical declarations here are to add an `_` prefix - `typedef struct _EC_KEY`. Again, just want to make sure we follow the current style.

Reply:
Current style in the Android crypto PAL is to use the same name in the struct name and typedef. See the `pal_rsa.h` header.

Reply:
I don't see any `struct`s declared in [`pal_rsa.h`](https://github.com/dotnet/runtime/blob/master/src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_rsa.h). But I see many matches for "typedef struct _" when I search the code base.

Reply:
Your link points to the Linux (not Android) `pal_rsa.h`. This is the example I was thinking of:
https://github.com/dotnet/runtime/blob/0026a202ab4087d237f6be8824dff322d3a7d6be/src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_rsa.h#L15

Reply:
Okay. Thanks.